### PR TITLE
Introduce --batch option for CLI

### PR DESCRIPTION
**Pull request by jablan**
https://memsource.tpondemand.com/entity/74406-orch-support-option-in-phrase-cli

### CODE REVIEW

**Comment by egze** [comment]
Can't really comment much on the implementation. Maybe let's get it live and test it in action?

**Comment by jablan** [comment]
@egze could you use just the binary from the branch, as sort of a QA run?

File: clients/cli/cmd/internal/push.go [hunk 1]
```diff
@@ -190,8 +194,7 @@ func (source *Source) Push(client *phrase.APIClient, waitForResults bool, branch
 				print.Failure("There was an error processing %s. Your changes were not saved online.", localeFile.RelPath())
 			}
 		} else {
-			fmt.Println("done!")
-			fmt.Printf("Check upload Id: %s, filename: %s for information about processing results.\n", upload.Id, upload.Filename)
+			print.NonBatchPrintf("done!\nCheck upload Id: %s, filename: %s for information about processing results.\n", upload.Id, upload.Filename)
```
Comment (forelabs) [inline clients/cli/cmd/internal/push.go:197]: I think we should have a json response here as well.
How about printing the whole `upload` object?

File: clients/cli/cmd/internal/upload_cleanup.go [hunk 2]
```diff
@@ -79,7 +85,7 @@ func UploadCleanup(client *phrase.APIClient, cmd *UploadCleanupCommand) error {
 			return err
 		}
 
-		fmt.Printf("%d key(s) successfully deleted.\n", affected.RecordsAffected)
+		print.Success(fmt.Sprintf("%d key(s) successfully deleted.\n", affected.RecordsAffected))
```
Comment (forelabs) [inline clients/cli/cmd/internal/upload_cleanup.go:88]: I think we should return json here, maybe the whole `affected` object?

**Review by forelabs — APPROVED** [review]

**Pull request merged**